CenterForOpenScience/ember-osf-web
# [IMLS/IA][ENG-2608] Update wording for registration creation modal

- Ticket: [ENG-2608]
- Feature flag: n/a

## Purpose
- Update wording in registration creation modal to mention registrations will be saved to Internet Archive

## Summary of Changes
- Update translation file
- Basically just re-opening #1145 

## Side Effects
- None, just wording changes

## QA Notes
- The wording in the `Almost done...` modal at the end of Registration creation (hitting `Submit` on the Draft Registration workflow) should mention that registrations are saved to Internet Archive


[ENG-2608]: https://openscience.atlassian.net/browse/ENG-2608
